### ISSUE TITLE
5v version of MB85RS

### ISSUE THREAD

**Issue by spoitras** [hackscribble/feedback#2]
The Fujitsu MB85RS64V seems to be a 5v variant of the chip you used in your design.  It could be a drop in replacement eliminating the need for the level conversion.  What do you think?

http://www.fujitsu.com/us/Images/MB85RS64V-DS501-00015-4v0-E.pdf

**Comment by hackscribble** [comment]
Yes, I would expect it to work with the library without needing level conversion.  Just use "MB85RS64" as the part number in the constructor.
